test/projects/file-generate/hack/generate-operator-bundle.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
```

Comment:
How do we keep these files in sync?

Reply:
which files ?

Reply:
Ah I see, this is purely a test project.